README.md

```diff
@@ -90,3 +90,39 @@ await mongoose.connect(url, defaultMongoOpts)
 const person = await PersonRepo.byID(id));
 const people = await PersonRepo.all({});
 ```
+
+## How to run a migration
+
+```shell
+yarn migrations init
+```
+
+The above command will initialize a `migrate-mongo-config.js` in the root directory and create a migration folder in your `src` folder.
+
+Now run
+
+```shell
+yarn migrations create <name>
+```
+
+The command helps to create a migration script in the `src -> migration` directory as below:
+
+```ts
+import { Db, MongoClient } from "mongodb";
+
+export async function up(db: Db, conn: MongoClient) {
+  // TODO write your migration here.
+  // Example:
+  // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: true}});
+}
+
+export async function down(db: Db, conn: MongoClient) {
+  // TODO write the statements to rollback your migration  (if possible)
+  // Example:
+  // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: false}});
+}
+```
+
+Import `migrateUp` or `migrateDown` function in your server (i.e index.ts) to apply migration.
```

Comment:
so by importing it my migration will be applied. And why my `server` or even `index.ts`